src/implementations/twig/storybook/ec/.storybook/main.js

```diff
@@ -3,12 +3,14 @@ const path = require('path');
 const stories = ['../../../**/!(eu*).story.js'];
 
 const addons = [
+  '@storybook/addon-docs',
```

Comment:
removing this will will remove the tabs docs tab

Reply:
not sure when you left this comment, if it's before the conversation we had on teams i guess we can close this.. :)